qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -156,6 +158,7 @@ def __init__(
         self._data_map_func = data_map_func or self_product
         self._paulis = paulis or ["Z", "ZZ"]
         self._alpha = alpha
+        self.entanglement = entanglement
```

Comment:
Is this necessary? This attribute should already be accessible via `self._entanglement`.